### ISSUE TITLE
Sampler in val dataloader

### ISSUE THREAD

**Issue by nicocheh** [jdb78/pytorch-forecasting#851]
Can i add a sampler for the validation_dataloader? I get an index problem if i try to predict

**Comment by ozanozyegen** [comment]
Yes, TimeSeriesDataSet object has a to_dataloader function that can take any pytorch sampler instance.
See `batch_sampler` parameter in [docs](https://pytorch-forecasting.readthedocs.io/en/stable/api/pytorch_forecasting.data.timeseries.TimeSeriesDataSet.html#pytorch_forecasting.data.timeseries.TimeSeriesDataSet.to_dataloader).
